calculator.py

```diff
@@ -46,15 +50,14 @@ def calculator():
             result = multiply(num1, num2)
             print(f"Result: {num1} * {num2} = {result}")
         elif choice == '4':
-            try:
-                result = divide(num1, num2)
-                print(f"Result: {num1} / {num2} = {result}")
-            except ValueError as e:
-                print(str(e))
+            result = divide(num1, num2)
```

Comment:
You removed the try catch error around the divide method call. Please add it back to prevent divide by 0 errors.